[2024-12-05T06:20:53.897+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-05T06:20:53.916+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_pipeline.save_to_postgres manual__2024-12-05T06:10:47.179856+00:00 [queued]>
[2024-12-05T06:20:53.927+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_pipeline.save_to_postgres manual__2024-12-05T06:10:47.179856+00:00 [queued]>
[2024-12-05T06:20:53.929+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 6
[2024-12-05T06:20:53.944+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): save_to_postgres> on 2024-12-05 06:10:47.179856+00:00
[2024-12-05T06:20:53.951+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3110) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-05T06:20:53.953+0000] {standard_task_runner.py:72} INFO - Started process 3111 to run task
[2024-12-05T06:20:53.953+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_pipeline', 'save_to_postgres', 'manual__2024-12-05T06:10:47.179856+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/weather_pipeline.py', '--cfg-path', '/tmp/tmp63kzhmm4']
[2024-12-05T06:20:53.954+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask save_to_postgres
[2024-12-05T06:20:54.000+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_pipeline.save_to_postgres manual__2024-12-05T06:10:47.179856+00:00 [running]> on host b86987f6f159
[2024-12-05T06:20:54.097+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Dung' AIRFLOW_CTX_DAG_ID='weather_pipeline' AIRFLOW_CTX_TASK_ID='save_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T06:10:47.179856+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T06:10:47.179856+00:00'
[2024-12-05T06:20:54.098+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-05T06:20:54.103+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-05T06:20:54.114+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-05T06:20:54.115+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_pipeline, task_id=save_to_postgres, run_id=manual__2024-12-05T06:10:47.179856+00:00, execution_date=20241205T061047, start_date=20241205T062053, end_date=20241205T062054
[2024-12-05T06:20:54.170+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-05T06:20:54.189+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-05T06:20:54.192+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
